#request (empty)
---
#response
geometry_msgs/Point position